src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/Utils/MdsValueChecker.java

```diff
@@ -31,15 +33,39 @@ public static void checkMdsValueOfAttributeWithoutDates(
         checkMdsValueInJsonObject(attribute, expectedValue, expectedIsVerified);
     }
 
+    public static void checkMdsValueOfAddress(
+            int index,
+            JSONObject attributes,
+            MatchingAddress matchingAddress
```

Comment:
It'd be worth switching round the attributes and matchingAddress parameters, so that they're in line with the order used in other MdsValueChecker methods.  For the same reason, I'd recommend renaming matchingAddress to expectedAddress.